NOTICE

```diff
@@ -1,27 +1,33 @@
 cloud-sdk-go
-Copyright 2021 Elasticsearch B.V.
+Copyright 2021-2022 Elasticsearch B.V.
```

Comment:
IANAL but wouldn't this just be `2022`, I'm guessing this file got automatically generated though.

Reply:
Yup. Even if I change it manually to be just `2022`, it goes back to `2021-2022` when running `make swagger`